src/saturn_engine/utils/tester/pipeline_test.py

```diff
@@ -58,6 +77,17 @@ def run_saturn_pipeline_test(
     expected_pipeline_results: list[dict] = [
         asdict(r) for r in pipeline_test.spec.pipeline_results
     ]
+
+    # Handle custom wildcard(__any__) fields in order to pass the assertion
+    for i in range(len(expected_pipeline_results)):
+        expected_pipeline_results[i], paths = replace_nodes(
```

Comment:
Why not find all `__any__` path and replace these path in the actual results? (Instead of replacing `__any__` to `__removed__`)